modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -395,6 +395,10 @@ void ParallelCoordinates::createOrUpdateProperties() {
 }
 
 void ParallelCoordinates::buildLineMesh() {
+    if (axes_.empty()) {
```

Comment:
seems strange to handle this here, should be handled in process

Reply:
Fixed

Reply:
Still no change though?